src/main/java/org/cyclops/integrateddynamics/core/evaluate/operator/Operators.java

```diff
@@ -780,6 +780,17 @@ public IValue evaluate(OperatorBase.SafeVariablesGetter variables) throws Evalua
                 variables -> ValueTypeString.ValueString.of(ValueTypes.CATEGORY_UNIQUELY_NAMED.getUniqueName(variables.getVariables()[0]))
             ).build());
 
+    /**
+     * Throw a custom error
+     */
+    public static final IOperator STRING_ERROR  = REGISTRY.register(OperatorBuilders.STRING_2.symbol("error").operatorName("string_error")
+            .inputType(ValueTypes.STRING).renderPattern(IConfigRenderPattern.SUFFIX_1_LONG)
+            .function(
+                (variables) -> {
+                    throw new EvaluationException(Component.translatable(variables.getValue(0, ValueTypes.STRING).getRawValue()));
```

Comment:
Is the `translatable` intentional here?
Could make sense to have that here indeed.

Reply:
I saw translatable getting used in other parts of the file to make a `MutableComponent` that the `EvaluationException` constructor expects, but if there's another way you'd prefer me make one, I can definitely change it.